stdlib/TOML/src/print.jl

```diff
@@ -104,6 +104,28 @@ function printvalue(f::MbyFunc, io::IO, value::TOMLValue)
     error("internal error in TOML printing, unhandled value")
 end
 
+function log2i(x::Base.BitUnsigned)
```

Comment:
we're about to add a very similar function under the name `top_set_bit`

Reply:
@oscardssmith could you point out the PR or issue? I cannot find this function in the source code, should I call this function in this PR, or I can just leave it as is for now?

Reply:
#47523

Reply:
OK maybe let me switch to that function after this PR merged then?

Reply:
merged